draft-ietf-tls-tls13.md

```diff
@@ -5069,6 +5072,58 @@ lacking a "server_name" extension by terminating the connection with a
 "missing_extension" alert.
 
 
+## Protocol Invariants
+
+This section describes invariants that TLS endpoints and intermediaries MUST
+follow. It also applies to earlier versions, which assumed these rules but did
+not document them.
+
+TLS is designed to be securely and compatibly extensible. Newer clients or
+servers, when communicating with newer peers, should negotiate the most
+preferred common parameters with downgrade protection.  In particular, it is a
```

Comment:
I'm not sure what the downgrade protection text means here.

Reply:
"is expected to negotiate the most-preferred common parameters, and any attempt to interfere with the negotiation MUST result in either failure of the interference or of the handshake" takes a slightly different approach.
Though that is rather duplicating the next sentence...